### DISCUSSION TITLE
blog/posts/isc2-ne-chapter-nov-2023

### DISCUSSION THREAD

**Discussion by giscus** [AlexHedley/blog#8] [General]
# blog/posts/isc2-ne-chapter-nov-2023

ISC2 Chapter Meetup

https://alexhedley.com/blog/posts/isc2-ne-chapter-nov-2023

**Comment by didley40** [comment]
Thanks for your comments, we worked hard as a team to get this event on and create an array of speakers covering a variety of subjects. We have a diverse membership and mixed it up. Glad liked the food and the chat as well as cool merch ! Please spread the word and come to our next event in the new year. Feel free to volunteer to talk about your own passions!

**Reply by AlexHedley** [reply]
Will do, and looking forward to the next one in the new year, great to have them in person again and I'd love to do a talk at some point, I'll ping you to discuss in the future.

**Comment by didley40** [comment]
Great let’s us know always keen and we are always looking for something different maybe not on trend but fundamentally a key area of interest or one that should be! 😊Thanks PaulBe inspired, happy and fulfilledIf you are not the intended recipient then please delete, please do not disclose the content or email addresses contained.On 12 Nov 2023, at 15:47, Alex Hedley ***@***.***> wrote:﻿
Will do, and looking forward to the next one in the new year, great to have them in person again and I'd love to do a talk at some point, I'll ping you to discuss in the future.
